hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAbfsInputStreamStatistics.java

```diff
@@ -160,7 +159,7 @@ public void testSeekStatistics() throws IOException {
       assertEquals("Mismatch in forwardSeekOps value", OPERATIONS,
           stats.getForwardSeekOperations());
       assertEquals("Mismatch in bytesBackwardsOnSeek value",
-          -1 * OPERATIONS * ONE_MB, stats.getBytesBackwardsOnSeek());
+          OPERATIONS * ONE_MB, stats.getNegativeBytesBackwardsOnSeek());
```

Comment:
why the name change? What does this statistic mean?

Reply:
bytesBackwardsOnSeek had negative increments and thus had a negative value but, since we have a restriction that counters can't increment negative values, I thought to have a negative of bytesBackwardsOnSeek counter so that we can have a counter to represent this value just in positive. Looks confusing I guess, should've come up with better naming :)